src/devices/tuya.ts

```diff
@@ -3235,6 +3235,26 @@ const definitions: DefinitionWithExtend[] = [
          * Initially wrapped in a try catch: https://github.com/Koenkk/zigbee2mqtt/issues/6313
          */
     },
+    {
+        zigbeeModel: ['TS0041A'],
+        fingerprint: [{manufacturerName: '_TYZB01_1xktopx6'}],
+        model: 'TS0041A',
+        vendor: 'Tuya',
+        description: 'Wireless switch with 1 button',
+        whiteLabel: [
+            {vendor: 'Marmitek', model: 'Push ME'},
```

Comment:
```suggestion
            {vendor: 'Marmitek', model: 'Push ME'},
```

Is there a better model number written on the device or product page?

Reply:
Unfortunately not. At least I did not find any at https://marmitek.com/en-de/products/push-me-zigbee-switch-scene-activation?country=DE